docs/arcadia-users-group/20230228-intro-to-python-1/lesson.ipynb

```diff
@@ -0,0 +1,884 @@
+{
```

Comment:
<p>do you mean variable <code>robot_name</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arcadia-Science/arcadia-computational-training/pull/50/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>